quasi_macros/src/lib.rs

```diff
@@ -419,11 +419,11 @@ fn expand_tts(cx: &ExtCtxt, sp: Span, tts: &[ast::TokenTree])
     p.quote_depth += 1;
 
     let cx_expr = p.parse_expr();
-    if !p.eat(&token::Comma) {
-        p.fatal("expected token `,`");
+    if !p.eat(&token::Comma).ok().unwrap() {
```

Comment:
The use of `.ok().unwrap()` panics if the result is `Err(_)`. In particular I dislike that `.ok()` discards the error first, so the reason of the failure can't be shown in the panic message.